homeconnect/api.py

```diff
@@ -127,8 +130,25 @@ def delete(self, endpoint):
     def get_appliances(self):
         """Return a list of `HomeConnectAppliance` instances for all
         appliances."""
+
+        appliances = []
+
         data = self.get(ENDPOINT_APPLIANCES)
-        return [HomeConnectAppliance(self, **app) for app in data["homeappliances"]]
+        for home_appliance in data["homeappliances"]:
+            found_matching = False
+            for appliance in self._appliances:
```

Comment:
This could be made less verbose if `self._appliance` were a dict, what do you think? Since it's a private attribute I wouldn't mind changing it.

Reply:
Good suggestion! I will change it!

Reply:
I don't think it's really possible to change it because then deleted appliances won't be removed from the list.

Reply:
Fixed!